eventdiscovery-sdk/src/main/res/layout/schedjoules_details_header.xml

```diff
@@ -16,8 +16,8 @@
                 android:fitsSystemWindows="true"
                 app:expandedTitleGravity="bottom|left"
                 app:expandedTitleTextAppearance="@style/SchedJoules_ExpandedTitle"
-                app:expandedTitleMarginStart="@dimen/schedjoules_activity_horizontal_margin"
-                app:expandedTitleMarginEnd="@dimen/schedjoules_activity_horizontal_margin"
+                app:expandedTitleMarginStart="17dp"
```

Comment:
I don't understand this. In your comment you state that the expanded title strangely gets aligned to 17dp instead of 18dp, but you explicitly set the margin to 17dp?! Am I missing something?